apprise/plugins/NotifyServerChan.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 Chris Caron <xuzheliang135@qq.com>
+# All rights reserved.
+#
+# This code is licensed under the MIT License.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files(the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions :
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import re
+import requests
+
+from ..common import NotifyType
+from .NotifyBase import NotifyBase
+from ..utils import validate_regex
+from ..AppriseLocale import gettext_lazy as _
+
+
+# Register at https://sct.ftqq.com/
+#   - do as the page describe and you will get the token
+
+# Syntax:
+#  serverchan://{access_token}/
+
+
+class NotifyServerChan(NotifyBase):
+    """
+    A wrapper for ServerChan Notifications
+    """
+
+    # The default descriptive name associated with the Notification
+    service_name = 'ServerChan'
+
+    # The services URL
+    service_url = 'https://sct.ftqq.com/'
+
+    # All notification requests are secure
+    protocol = 'serverchan'
```

Comment:
if protocol `==` `secure_protocol` it's a bit redundant.  Just remove the `protocol=` entry.